front-end/src/components/newsfeed-view/news-list/news-card/FacebookNewsCard.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const FacebookNewsCard = ({data}) => (
+    <div className="news_card facebook">
```

Comment:
inconsistent case. I believe we should stick with hyphenated name for css classes. so `news-card` instead.